src/js/player.js

```diff
@@ -448,17 +451,16 @@ class Player extends Component {
     // Ensure the right class is still on the player for the style element
     this.addClass(idClass);
 
-    // Create the width/height CSS
-    var css = `.${idClass} { width: ${width}px; height: ${height}px; }`;
-    // Add the aspect ratio CSS for when using a fluid layout
-    css += `.${idClass}.vjs-fluid { padding-top: ${ratioMultiplier  * 100}%; }`;
+    stylesheet.setTextContent(this.styleEl_, `
```

Comment:
I'm using one template string here to add both the previous css selectors to the style element.